From 339e3d1d766fd64a373cd4c4cdc2a544e36306be Mon Sep 17 00:00:00 2001
From: NIIBE Yutaka <gniibe@fsij.org>
Date: Tue, 14 May 2024 15:48:45 +0900
Subject: [PATCH] dirmngr: Fix a call of calloc.

* dirmngr/ldap-parse-uri.c (ldap_parse_uri): Fix arguments.

--

Cherry pick master commit of:
	14534e72e1135a0323b896339bd1f508a592a7be

Signed-off-by: NIIBE Yutaka <gniibe@fsij.org>
---
 dirmngr/ldap-parse-uri.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dirmngr/ldap-parse-uri.c b/dirmngr/ldap-parse-uri.c
index 573bcc77f..5856c1b4e 100644
--- a/dirmngr/ldap-parse-uri.c
+++ b/dirmngr/ldap-parse-uri.c
@@ -162,7 +162,7 @@ ldap_parse_uri (parsed_uri_t *purip, const char *uri)
 
   if (password)
     {
-      puri->query = calloc (sizeof (*puri->query), 1);
+      puri->query = calloc (1, sizeof (*puri->query));
       if (!puri->query)
         {
           err = gpg_err_code_from_syserror ();
